### ISSUE TITLE
Alias to spam does not work from Mailcow web UI

### ISSUE THREAD

**Issue by DamianPereira** [mailcow/mailcow-dockerized#1972]
**Describe the bug**

I'm trying to setup my mailbox so that certain addresses go throguh, but everything else goes to spam. so info@domain.com, sales@domain.com, and stuff like that goes to inbox, but adsf@domain.com goes straight to spam folder.

But if I create a catch-all alias and set it up as learn as spam, mailcow UI says "Goto address spam@localhost is invalid" when trying to activate the alias (togge the checkbox, and try to activate/desactivate). Even creating an alias such as "thisisspam@domain.com" and clicking learn as spam, results in the same dialog.

**System**
 - OS: ubuntu 18.04
 - Browser: Chrome

**Comment by andryyy** [comment]
Something is not up to date in your code, I think.

![grafik](https://user-images.githubusercontent.com/2972950/47842746-e5dfb180-ddbd-11e8-8d09-9cc1cabd48e1.png)

Is this an older mailcow installation? Can you try to run `git fetch origin master ; git checkout origin/master data/web`?

**Comment by DamianPereira** [comment]
I ran the update.sh script before creating the issue, but I will try that and report back. Did you make sure to toggle and activate it? I only get the error that way. And the messages are rejected silently instead of being sent into Spam, do they reach the spam folder in your install?

**Comment by andryyy** [comment]
I don’t really understand your sentence here. 

Do you create an inactive alias, go back to edit it and then it fails? Please describe the steps.

**Comment by andryyy** [comment]
It says the messages are learned as spam. It does not say they are being delivered to your spam folder. They are just learned as spam and discarded.

**Comment by DamianPereira** [comment]
Sorry for the delay, the update process broke mysql and I had to reinstall everything, I installed everything from scratch and the issue still persists. The steps to reproduce are the following:

1. Create a new alias, the alias field does not matter, I used asdf@domain.com, select learn as spam and active
2. Toggle the checkbox to the left of the new filter
3. Go to actions, select activate or deactivate (both produce the error)
4. Get 2 popups in the bottom right corner, a green one that says "Changes to alias address [alias name] have been saved" and a red one that says: "Goto address spam@localhost is invalid"

> It says the messages are learned as spam. It does not say they are being delivered to your spam folder. They are just learned as spam and discarded.

I thought the messages were supposed to reach the spam folder, or at least reach the quarantine. But if it's the intented behavior then it's not part of the issue and maybe the error is just from the UI and everything works ok behind it.
